internal/controllers/plans.go

```diff
@@ -0,0 +1,39 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/cyverse/QMS/internal/model"
+	"github.com/labstack/echo"
+)
+
+// swagger:route GET /plans plans listPlans
+// Returns a List all the plans
+// responses:
+//   200: plansResponse
+//   404: RootResponse
```

Comment:
```suggestion
// swagger:route GET /plans plans listPlans
//
// List Plans
//
// Returns a List of all the plans.
//
// responses:
//   200: plansResponse
//   404: RootResponse
```

See the comment for `GET /plans/{planID}` for an explanation.